docs/DELTA.md

```diff
@@ -86,4 +86,122 @@ if (auction?.status === 'EXECUTED') {
 }
 ```
 
-#### A more detailed example of Delta Order usage can be found in [examples/delta](./src/examples/delta.ts)
\ No newline at end of file
+#### A more detailed example of Delta Order usage can be found in [examples/delta](./src/examples/delta.ts)
+
+
+
+
+---------------------------------------------------
+
+## Crosschain Delta Orders
+
+In case you want to perform a crosschain swap, the process is a little more involved.
+
+### 1. Construct an SDK object
+
+```ts
+const SRC_CHAIN_ID = 1 // Ethereum
+const DEST_CHAIN_ID = 10; // Optimism
+
+const account = userAddress;
+const deltaSDK = constructSimpleSDK(
+  {chainId: SRC_CHAIN_ID, axios},
+  {
+    ethersProviderOrSigner: provider, // JsonRpcProvider
+    EthersContract: ethers.Contract,
+    account,
+  });
+  // for usage with different web3 provider libraries refer to the main [README](./README.md)
+```
+
+### 2. Check which tokens are available on the destination chain.
+
+A limited list of tokens are available in Across, the service facilitating crosschain bridging
+
+```ts
+const bridgeInfo = await deltaSDK.getBridgeInfo();
+
+const tokensAvailableForBridging = bridgeInfo[SRC_CHAIN_ID]?.[DEST_CHAIN_ID]
+```
+
+
+### 3. Request prices for a Token pair
+
+```ts
+const DAI_TOKEN = '0x6b175474e89094c44da98b954eedeac495271d0f';
+
+const USDC_TOKEN_ON_DEST_CHAIN = '0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85'; // available for bridging on Optimism
+const amount = '1000000000000'; // in wei
+
+const deltaPrice = await deltaSDK.getDeltaPrice({
+  srcToken: DAI_TOKEN,
+  destToken: USDC_TOKEN_ON_DEST_CHAIN,
+  destChainId: DEST_CHAIN_ID, // required to get deltaPrice for a Crosschain Order
+  amount,
+  userAddress: account,
+  srcDecimals: 18,
+  destDecimals: 18,
+  // partner: "..." // if available
+});
+```
+
+
+### 4. Approve srcToken for DeltaContract
+
+```ts
+const tx = await deltaSDK.approveTokenForDelta(amount, DAI_TOKEN);
+await tx.wait();
+```
+
+Alternatively sign Permit (DAI or Permit1) or Permit2 TransferFrom with DeltaContract as the verifyingContract
+
+```ts
+const DeltaContract = await deltaSDK.getDeltaContract();
+
+// values depend on the Permit type and the srcToken
+const signature = await signer._signTypedData(domain, types, message);
+```
+
+See more on accepted Permit variants in [Velora documentation](https://developers.velora.xyz/api/velora-api/velora-delta-api/build-a-delta-order-to-sign#supported-permits)
```

Comment:
would it be feasible to integrate permit logic to into the SDK in the next iterations?

Reply:
Not feasible at all. Remember the complex logic to get Token's data for permit, how many tokens don't follow the standard for domain, version, nonce and other methods.

Reply:
outcome after DM convo: permit2 seems worth embedding